### DISCUSSION TITLE
There seems to be an issue with repositories using core.worktree.

### DISCUSSION THREAD

**Discussion by CapinWinky** [libgit2/libgit2#5863] [Q&A] [answered]
I recently tested numerous major Windows git GUIs and found that of the 10 tested, only Git Extensions correctly handled a repository that used `core.worktree` to move the main working tree outside of the repository path. It was suggested to me that the other GUIs may all be users of libgit2 and that the fault may lie with this implementation.

More detailed information can be found in my reddit post: [Here](https://www.reddit.com/r/git/comments/nak469/coreworktree_support_completely_lacking_in_almost/?ref=share&ref_source=link) with application specific failure mode details linked from there in another reddit post.

**Comment by ethomson** [comment, marked as answer]
Some of those GUIs use libgit2 entirely (for example, GitKraken), some of them use the git CLI only (for example, Tower), and some of them use a mix (for example, Tortoise Git).

You'll really need to discuss the functionality with the authors of the packages, who can then identify what limitations are in their software, versus libgit2.

**Comment by iwl** [comment]
Hi, I solve that problem with Hardlinks mklink /H and Directory Junctions mklink /J in git workdir to the workdir without any git in.
